index.js

```diff
@@ -57,10 +60,27 @@ module.exports = function (options) {
 		}
 
 		try {
-			var errors = checker.checkString(file.contents.toString(), file.relative);
+			var fixResults;
+			var errors;
+			var changed = false;
```

Comment:
This isn't detecting if the original file changed, but rather if the process of running jscs on the file produced a change. Not running on the original file would limit the input optimization. This check was meant to allow the next step in the gulp pipeline to further optimize. For instance, if you have a large number of files that haven't changed, but you add or change a jscs rule in your config this would have allowed the filtering of just the few files affected by the new rule to be written to disk vs writing the entire directory again. But I could be not fully understanding all the ways gulp-changed could be used.